async_remote.md

```diff
@@ -0,0 +1,27 @@
+[other sections soon to follow]
+
+# Knowledge transfer
+
+## Wiki checklist
+
+### Index
+  List and describe all the tools, services and things used in the project.
+  
+### Glossary
+  List and describe most important and required-to-know concepts and terminology used in the project.
+  This section should usually provide newly onboarded team members with important know-how on what means what in the project.
+  Weird acronyms? Overloaded terms which mean different things in normal life? Proper names (_pl. nazwy wlasne_)? They all should be here.
```

Comment:
I think at some point it deserves it's own part of the style-guide, but since it's now part of the introductory take on using Wikis, I'd be happy to see more emphasis on estabilishing an ubiquitous language here. E.g.

"Use the glossary to establish a well-understood language and terms, so that all team members understand any given term in similar way."

The purpose of the glossary, for me, is not only to _explain_ the terms (like an encyclopedia), but also put spotlight on whether the terms are well-understood and check if there are no spurrious duplicates or terms raising eyebrows.

Reply:
It's especially important with terms that have a common sense counterpart that might be misleading – e.g. in IOP we have Sellers (that could be described as affiliates in different systems, but even there the definition is a bit imprecise) who do not sell anything of their own.

Reply:
Fair enough, I agree. :)